src/credential/credential-internal.ts

```diff
@@ -19,7 +19,7 @@ import fs = require('fs');
 import os = require('os');
 import path = require('path');
 
-import { GoogleOAuthAccessToken, Credential } from './credential';
+import { GoogleOAuthAccessToken, Credential } from './google-oauth-access-token';
```

Comment:
Lets call it `credential-interfaces.ts` since it has more than just the access token type.